src/hooks/useValidationClass.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+export default function useValidationClass({
+  formElementsValid,
+  displayValidationMessage,
+  validClassName,
+  invalidClassName,
+}: {
+  formElementsValid: boolean
+  displayValidationMessage: boolean
+  validClassName: string
+  invalidClassName: string
+}): {
+  validationClassName: string
+  valid: boolean
+} {
+  return React.useMemo(() => {
+    if (!formElementsValid) {
```

Comment:
```suggestion
    if (formElementsValid) {
```
Should be the other way around, no?

Reply:
Also fixed that badness